provisioning/security/security-provider/src/main/java/com/microsoft/azure/sdk/iot/provisioning/security/SecurityProviderSymmetricKey.java

```diff
@@ -106,43 +105,6 @@ public String getRegistrationId() throws SecurityProviderException
         return registrationId;
     }
 
-    /**
-     * Retrieves the SSL context loaded with trusted certs. In case of X509 SSL context shall be loaded with complete chain
-     * all the way till the leaf along with its private key.
-     *
-     * @return The SSLContext relevant to the flow
-     * @throws SecurityProviderException If ssl context could not be generated for any of the reason
-     */
-    @Override
-    public SSLContext getSSLContext() throws SecurityProviderException
```

Comment:
This implementation was shared with the SecurityProviderTpm class, so I moved it up to the more abstract SecurityProvider class while stripping out the reading of certs from source